pkgs/by-name/be/beszel/package.nix

```
@@ -4,18 +4,18 @@
   fetchFromGitHub,
   nix-update-script,
   buildNpmPackage,
-  fetchpatch,
+  go_1_24,
 }:
 
-buildGoModule rec {
+buildGoModule.override { go = go_1_24; } rec {
```

```suggestion
buildGo124Module rec {
```
Or is there some reason to prefer the override solution?

---

You are right! I checked if the `buildGo124Module` was available, but I am running the stable release and forgot to check the unstable branch.